packages/framework/src/Migrations/Version20190821060201.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20190821060201 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('ALTER TABLE products ADD selling_from_date_time TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL');
```

Comment:
Would not that be easier to update type of current columns? I was looking at the entity and there are mixed names "sellingFrom" and "sellingDateTimeFrom" or something like that and I think you are also little confused how to use that right way.

Reply:
Why are you updating this? I know that selling denied is better in exact time, but it is not part of this US, right?

Reply:
Change the type of column is considered a BC break – https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md#database-migrations 

You're right the naming is confusing, but the idea is: just the name of the protected property is changed and the rest of the application still works as before (still use sellingFrom)

And why I'm updating this. Consider someone in Phoenix on 21.02.2019 20:00 (UTC-7) sets SellingFrom to another day (22.02.2019).

What date should be saved in the database that is in UTC? 

Let's say I save exactly what user enters – 22.02.2019. 
Server time is at the moment of change 22.02.2019 03:00 and running recalculation make the product visible immediately. Not good.

Let's say I convert Date(Time) to UTC before saving.
Convert the DateTime returned from Symfony Form `2019-02-22 00:00:00.0 America/Phoenix (-07:00)` to UTC. 
That is `2019-02-22 07:00:00.0 UTC (+00:00)` but into the database i still save only 22-02-2019. And we're at the exactly same spot as before.

Having the time saved solves this problem as I can now save and compare with exact time moment.

But the time currently should not be used to set product visible at the exact time of the day (ie. at 13:25) because the recalculations are not executed so often.

Reply:
Ok good point, but I consider changing date to date_time as exception from our BC break same as we allow changing array to iterable. It will still behave same way.

Reply:
Hmm... it makes sense... 
the tests will not fail because on the testing database are run DB migrations before the testing starts (when it's run with Phing).

The `./phing checks` target will fail as the database schema is not satisfying ORM (Phing target `db-check-schema`)

But apart from that everything seems to be working.

Reply:
fixed in 610c3c3df026edf41e47854a89938bb9ecacc0fb and 60713fae9f68e33d67c5fd8867d71f1b8adf680a